app/models/request.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+# Model for requests in Symphony
+class Request
+  attr_reader :record
+
+  def initialize(record)
+    @record = record
+  end
+
+  def key
+    record['key']
+  end
+
+  def status
+    fields['status']
+  end
+
+  def catkey
+    fields['item']['fields']['bib']['key']
+  end
```

Comment:
Note, I haven't done an exhaustive look over the designs, in particular with the different flavors of dates that we want to use vs the name that's used in the API.  I figured this would be a good thing to pair up on as we start getting the data into the UI.